lib/assertions.js

```diff
@@ -766,39 +766,65 @@ module.exports = (expect) => {
     );
   });
 
-  expect.addAssertion('<array-like> [not] to contain <any+>', function (
-    expect,
-    subject
-  ) {
-    const args = Array.prototype.slice.call(arguments, 2);
-    expect.withError(
-      () => {
-        args.forEach((arg) => {
-          expect(
-            subject &&
-              Array.prototype.some.call(subject, (item) =>
-                expect.equal(item, arg)
-              ),
-            '[not] to be truthy'
-          );
-        });
-      },
-      (e) => {
-        expect.fail({
-          diff:
-            expect.flags.not &&
-            ((output, diff, inspect, equal) =>
-              diff(
-                subject,
-                Array.prototype.filter.call(
+  expect.addAssertion(
+    [
+      '<array-like> to [only] contain <any+>',
+      '<array-like> [not] to contain <any+>',
+    ],
+    function (expect, subject) {
+      const args = Array.prototype.slice.call(arguments, 2);
+
+      expect.withError(
+        () => {
+          args.forEach((arg) => {
+            expect(
+              subject &&
+                Array.prototype.some.call(subject, (item) =>
+                  expect.equal(item, arg)
```

Comment:
I wonder if we should use `expect(item, 'to equal', arg)` here so it would respect an assertion "overloading" of `to equal` using `addAssertion` such as https://github.com/unexpectedjs/unexpected-function-equality

... OTOH that'd be slower and maybe even an antipattern 🤷‍♂️

Reply:
It would be a little slower, but not much. Let's just wait with that change till we need it.

Reply:
I think I have to `301` to @sunesimonsen here :-)

Reply:
I agree, I'm fine with the current impl.